From 1957bb5113a2bf02c7ecf0815125eec04b9476d2 Mon Sep 17 00:00:00 2001
From: Bruno Haible <bruno@clisp.org>
Date: Sat, 16 Apr 2022 03:01:51 +0200
Subject: sigsegv: Fix compilation error on arceb CPUs.

Reported by Fabrice Fontaine <fontaine.fabrice@gmail.com> in
<https://lists.gnu.org/archive/html/bug-gnulib/2022-04/msg00028.html>.

* m4/stack-direction.m4 (SV_STACK_DIRECTION): Treat the 'arc' variants
like 'arc'.

[Retrieved (and backported) from:
https://git.savannah.gnu.org/cgit/gnulib.git/commit/?id=1957bb5113a2bf02c7ecf0815125eec04b9476d2]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 ChangeLog             | 8 ++++++++
 m4/stack-direction.m4 | 4 ++--
 2 files changed, 10 insertions(+), 2 deletions(-)

diff --git a/m4/stack-direction.m4 b/m4/stack-direction.m4
index 1df823e..924c5de 100644
--- a/m4/stack-direction.m4
+++ b/m4/stack-direction.m4
@@ -1,4 +1,4 @@
-# stack-direction.m4 serial 7
+# stack-direction.m4 serial 11
 dnl Copyright (C) 2002-2021 Free Software Foundation, Inc.
 dnl This file is free software, distributed under the terms of the GNU
 dnl General Public License.  As a special exception to the GNU General
@@ -18,7 +18,7 @@ AC_DEFUN([SV_STACK_DIRECTION],
       a29k | \
       aarch64* | \
       alpha* | \
-      arc | \
+      arc | arceb | arc32 | arc64 | \
       arm* | strongarm* | xscale* | \
       avr | avr32 | \
       bfin | \
-- 
cgit v1.1

